### DISCUSSION TITLE
Playback does not work when using latest Widevine

### DISCUSSION THREAD

**Discussion by amyren1966** [CastagnaIT/plugin.video.netflix#1156] [General]
This is on Zgemma H9 Twin Enigma 2 tuner, Kodi 18.9 on OpenPli 8.0. netflix plugin is version 1.12.4
When setting up and succed with login it will ask to downloading Widevine CDM the first time I try to play anything from Netflix.
After this is downloaded and installed it wont play anything.

I had an old kodi install (from a few months back), which I did make a copy of the .kodi folder.
When I copy the cdm folder from the old backup and replace the one in my latest install then it works.

Is there something changed in the latest Widevine CDM that makes the plugin not to playback?

**Comment by cyr06** [comment]
My first time with kodi 18.9 and this add-on.
Installation goes fine. I have access to my account
Then it downloads Widevine CDM and... no playback, no error, just the list of episodes
I don't have any former copy to test but I tried twice from clean install (LE)
If someone can help ;)
thank you

**Reply by amyren1966** [reply]
That is the same sympthom I got, which I resolved by using an older copy of the cdm folder.
I dont know what platform you are running Kodi on, so I dont know if the files are similar for all platforms. But I have created a zip archive of my working cdm folder and put it temporary on my google drive.
Unzip it on your computer and tranfer to your .kodi folder on your device.
I think not all files are needed, I think you could try first with just replacing the libwidevinecdm.so file.
Link to the [archive](https://drive.google.com/file/d/1NLWOBNolqYLi2tO6Ha_vBXIBTSqls2nJ/view?usp=sharing)

**Reply by amyren1966** [reply]
Read [this info](https://www.gitmemory.com/issue/emilsvennesson/script.module.inputstreamhelper/59/521273546) about what I think is a similar issue with inputstreamhelper using an old version of widevine.
If you are on a windows x64 platform, the post will also contain a direct link to download a previous version of the archive. For Windows it will be libwidevinecdm.dll and not libwidevinecdm.so as I mentioned in my previous post.
That same post also show a link to a list of version numbers for widewine. Just replace the version number in the download link to get a specific version.  It looks like the latest version was released 30.march, and before that 22.march, then back to 2029 with version 4.10.1582.2.
Different architectures are available to, just try replacing "win-x64" with win-ia32", "linux-x64" or "linux-ia32" (not all versions available for linux-ia32)
For ARM devices, there does not seem to be a simple download link for the needed files. Thats why it will download the chromeos image and extracts the needed files from there. I havent found an easy way to do this manually. Luckily in my case, I had that old copy.

Perhaps CastagnaIT can enlighten us some more here (or correct me in case I got it all wrong:)

**Reply by amyren1966** [reply]
Update: Aperently someone solved the video not playing problem by following this tip. (found on a satelitte forum)

_you should open the Inputstream Helper addon (opening the addon will actually open its settings page), where you should enable the option called "Disable"._

**Comment by cyr06** [comment]
I think this is a duplicate issue with that one
https://github.com/CastagnaIT/plugin.video.netflix/issues/1154

**Reply by amyren1966** [reply]
Yes, it looks like the same issue.
Did you resolve your problem then, and if so, what solution did you try?

**Reply by cyr06** [reply]
not really because I don't have a previous version to rollback

**Reply by amyren1966** [reply]
A fix have been released.
You must update your plugin Inputstreamhelper to version 0.5.3
If your repo only shows plugin 0.5.2, you can download the 0.5.3 zip file from here and use the install from zip file option.
https://github.com/emilsvennesson/script.module.inputstreamhelper/releases

Then open the inputstreamhelper settings and select (re)install widevine libraries. Then an older version of the chromeOS image will be downloaded and this should fix the problem.

**Reply by NiklasTheis** [reply]
After updating my plugin Inputstreamhelper to 0.5.5 I was able so reinstall widevine. This then installed the previous widevine Version and now it is working super smooth.
